test/e2e/apps/rc.go

```diff
@@ -362,7 +380,10 @@ var _ = SIGDescribe("ReplicationController", func() {
 					foundRc = true
 				}
 			}
-			framework.ExpectEqual(foundRc, true)
+			if !foundRc {
+				framework.Logf("Got unexpected replication controller list %v", rcs.Items)
```

Comment:
Following the suggestion before I logged this list too and made the message a bit more descriptive